packages/amplify-app/src/xcodeHelpers.js

```diff
@@ -0,0 +1,50 @@
+const xcode = require('xcode');
+const path = require('path');
+const fs = require('fs-extra');
+
+async function getXcodeProjectDir() {
+  const EXTENSION = '.xcodeproj';
+  const files = fs.readdirSync(process.cwd());
+  const targetFiles = files.filter(function(file) {
+    return path.extname(file).toLowerCase() === EXTENSION;
+  });
+  let projDir;
+  if (targetFiles.length) {
+    projDir = path.join(path.join(process.cwd(), targetFiles[0]), '/project.pbxproj');
```

Comment:
`path.join` can take any number args
change this
```js
projDir = path.join(path.join(process.cwd(), targetFiles[0]), '/project.pbxproj');
```

to 
```js
projDir = path.join(process.cwd(), targetFiles[0], '/project.pbxproj');
```

Reply:
changed